src/Components/ContainerItem/ContainerItem.js

```diff
@@ -307,6 +313,127 @@ export default function ContainerItem({
+													<div className="filters_block">
+														<Heading
+															align="center"
+															weight="normal"
+															size={6}
+															style={{
+																display: 'flex',
+																alignItems: 'center',
+																justifyContent: 'space-between',
+																margin: 0,
+															}}
+														>
+															<span style={{ position: 'relative' }}>
+																Filter
+																<div className="tooltip_block">
+																	<img
+																		className="input_icon"
+																		src="/img/icons/info_circle.svg"
+																		height={18}
+																		width={18}
+																		alt="info"
+																	/>
+																	<div className="tooltip">Flexible search allows you to use any attributes to search through the attribute-operation-value formula, you can use parameters such as FilePath, FileName, etc.</div>
+																</div>
+															</span>
+															<Button
+																color="primary"
+																size="small"
+																onClick={() => {
+																	let filtersTemp = [ ...filters ];
+																	filtersTemp.push({
+																		key: '',
+																		match: 'MatchStringEqual',
+																		value: '',
+																	});
+																	setFilters(filtersTemp);
+																}}
+																disabled={isLoadingObjects}
+															>
+																Add filter
+															</Button>
+														</Heading>
+														{filters.map((filterItem, indexFilter) => (
+															<div className="filter_item" key={indexFilter}>
+																<Form.Control>
+																	<Form.Input
+																		size='small'
+																		placeholder="Attribute"
+																		value={filterItem.key}
+																		onChange={(e) => {
+																			let filtersTemp = [ ...filters ];
+																			filtersTemp[indexFilter].key = e.target.value;
+																			setFilters(filtersTemp);
+																		}}
+																		disabled={isLoadingObjects}
+																	/>
+																</Form.Control>
+																<Form.Control className="select_wrapper">
+																	<Form.Select
+																		size='small'
+																		value={filterItem.match}
+																		onChange={(e) => {
+																			let filtersTemp = [ ...filters ];
+																			filtersTemp[indexFilter].match = e.target.value;
+																			setFilters(filtersTemp);
+																		}}
+																		disabled={isLoadingObjects}
+																	>
+																		{[
+																			{ value: 'MatchStringEqual', title: '=' },
+																			{ value: 'MatchStringNotEqual', title: '≠' },
+																			{ value: 'MatchNumGT', title: '>' },
+																			{ value: 'MatchNumGE', title: '≥' },
+																			{ value: 'MatchNumLT', title: '<' },
+																			{ value: 'MatchNumLE', title: '≤' },
```

Comment:
Please drop GT/GE/LT/LE since they're not supported. But add prefix-based ("starts with"/"has prefix") one, it's useful.